include/inexor/vulkan-renderer/world/indentation.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cstdint>
+
+namespace inexor::vulkan_renderer::world {
+
+class Indentation {
+public:
+    static constexpr std::uint8_t MAX = 8;
+
+private:
+    std::uint8_t m_start = 0;
+    std::uint8_t m_end = Indentation::MAX;
+
+public:
+    Indentation() = default;
+    Indentation(const std::uint8_t start, const std::uint8_t end) noexcept;
+    explicit Indentation(const std::uint8_t uid) noexcept;
+    bool operator==(Indentation const &rhs) const;
+    bool operator!=(Indentation const &rhs) const;
```

Comment:
```suggestion
    bool operator==(const Indentation &rhs) const;
    bool operator!=(const Indentation &rhs) const;
```
Not sure what happened here?